hatchet_sdk/hatchet.py

```diff
@@ -47,40 +58,43 @@ def inner(cls) -> WorkflowMeta:
         cls.concurrency_expression = concurrency
         # Define a new class with the same name and bases as the original, but
         # with WorkflowMeta as its metaclass
-        return WorkflowMeta(cls.name, cls.__bases__, dict(cls.__dict__))
+
+        ## TODO: Figure out how to type this metaclass correctly
+        return WorkflowMeta(cls.name, cls.__bases__, dict(cls.__dict__))  # type: ignore[no-untyped-call]
 
     return inner
 
 
 def step(
     name: str = "",
     timeout: str = "",
-    parents: List[str] | None = None,
+    parents: list[str] | None = None,
     retries: int = 0,
-    rate_limits: List[RateLimit] | None = None,
-    desired_worker_labels: dict[str:DesiredWorkerLabel] = {},
-):
+    rate_limits: list[RateLimit] | None = None,
+    desired_worker_labels: dict[str, DesiredWorkerLabel] = {},
```

Comment:
😭 python typing is so hard ... why didn't this complain!

Reply:
😂 yeaahh....